pyproject.toml

```
@@ -16,7 +16,7 @@ dependencies = [
     "build>=0.7.0",
     "cachetools>=4.1",
     "click>=4.0",
-    "cookiecutter>=2.1.1,<3.0",
+    "cookiecutter==2.2.0",
```

Let's perhaps drop a comment

```suggestion
    "cookiecutter==2.2.0",  # 2.3 onwards depends on rich, which we want to avoid
```

?

---

Good idea